wp-content/plugins/sqlite-database-integration/wp-includes/mysql-parser/MySQLLexer.php

```diff
@@ -0,0 +1,9061 @@
+    public const DOT_IDENTIFIER = 799;
+    // Other
+    public const INVALID_INPUT = 800;
+    public const LINEBREAK = 801;
+    // Missing from the generated lexer and added manually
+    public const START_SYMBOL = 802;
+    public const UNLOCK_SYMBOL = 803;
+    public const CLONE_SYMBOL = 804;
+    public const GET_SYMBOL = 805;
+    public const ASCII_SYMBOL = 806;
+    public const BIT_SYMBOL = 807;
+    public const BUCKETS_SYMBOL = 808;
+    public const COMPONENT_SYMBOL = 809;
+    public const NOW_SYMBOL = 810;
+    public const DEFINITION_SYMBOL = 811;
+    public const DENSE_RANK_SYMBOL = 812;
+    public const DESCRIPTION_SYMBOL = 813;
+    public const FAILED_LOGIN_ATTEMPTS_SYMBOL = 814;
+    public const FOLLOWING_SYMBOL = 815;
+    public const GROUPING_SYMBOL = 816;
+    public const GROUPS_SYMBOL = 817;
+    public const LAG_SYMBOL = 818;
+    public const LONG_SYMBOL = 819;
+    public const MASTER_COMPRESSION_ALGORITHM_SYMBOL = 820;
+    public const NOT2_SYMBOL = 821;
+    public const NO_SYMBOL = 822;
+    public const REFERENCE_SYMBOL = 823;
+    public const RETURN_SYMBOL = 824;
+    public const SPECIFIC_SYMBOL = 825;
+    public const AUTHORS_SYMBOL = 826;
+    public const ADDDATE_SYMBOL = 827;
+    public const CONCAT_PIPES_SYMBOL = 828;
+    // Unused in this class but present in MySQLParser, hmm
+    public const ACTIVE_SYMBOL = 829;
+    public const ADMIN_SYMBOL = 830;
+    public const EXCLUDE_SYMBOL = 831;
+    public const INACTIVE_SYMBOL = 832;
+    public const LOCKED_SYMBOL = 833;
+    public const ROUTINE_SYMBOL = 834;
+    public const UNTIL_SYMBOL = 835;
+    public const ARRAY_SYMBOL = 836;
+    public const PASSWORD_LOCK_TIME_SYMBOL = 837;
+    public const NCHAR_TEXT = 838;
+    public const LONG_NUMBER = 839;
+    public const ULONGLONG_NUMBER = 840;
+    public const CUME_DIST_SYMBO = 841;
+    public const CUME_DIST_SYMBOL = 842;
+    public const FOUND_ROWS_SYMBOL = 843;
+    public const CONCAT_SYMBOL = 844;
+    public const OVER_SYMBOL = 845;
+    public const IO_THREAD_SYMBOL = 846;
+    public const REPLICA_SYMBOL = 847;
+    public const CONSTRAINTS_SYMBOL = 848;
+
+    public const EOF = -1;
+
+    protected $input;
+    protected $c; // Current character.
+    protected $n; // Next character.
+    protected $position = 0;
+    protected $token;
+    protected $text = '';
+    protected $channel = self::DEFAULT_TOKEN_CHANNEL;
+    protected $type;
+    protected $tokenInstance;
+    protected $serverVersion;
+    protected $sqlModes;
+
+    protected const DEFAULT_TOKEN_CHANNEL = 0;
+    protected const HIDDEN = 99;
+
+    public function __construct(string $input, int $serverVersion = 80000, int $sqlModes = 0)
+    {
+        $this->input = $input;
+        $this->serverVersion = $serverVersion;
+        $this->sqlModes = $sqlModes;
+    }
+
+    const PipesAsConcat = 1;
+    const HighNotPrecedence = 2;
+    const NoBackslashEscapes = 4;
+    public const ANSI_QUOTES = 8;
```

Comment:
this is an interesting bit. it appears like this was arbitrarily chosen as the only escaping mode? given an arbitrary enum value?